### PR TITLE
[WIP] Add the ability to right click in the VisualScript editor to open the generic search

### PR DESCRIPTION
**Pull request by Jummit**

### CODE REVIEW

**Comment by fire** [comment]
This was in the original patch for VS, but there was some complaints about it. I don't remember the exact reasons. Please comment.

**Comment by fire** [comment]
I am in favour of it.

File: modules/visual_script/visual_script_editor.cpp [hunk 1]
```diff
@@ -42,6 +42,8 @@
 #include "visual_script_func_nodes.h"
 #include "visual_script_nodes.h"
 
+#include <iostream>
```
Comment (fire) [inline modules/visual_script/visual_script_editor.cpp]: Is this required?

File: modules/visual_script/visual_script_editor.cpp [hunk 2]
```diff
@@ -3699,6 +3711,7 @@ static void register_editor_callback() {
 	ED_SHORTCUT("visual_script_editor/cut_nodes", TTR("Cut Nodes"), KEY_MASK_CMD + KEY_X);
 	ED_SHORTCUT("visual_script_editor/paste_nodes", TTR("Paste Nodes"), KEY_MASK_CMD + KEY_V);
 	ED_SHORTCUT("visual_script_editor/edit_member", TTR("Edit Member"), KEY_MASK_CMD + KEY_E);
+	ED_SHORTCUT("visual_script_editor/insert_comment", TTR("Insert Comment"), KEY_MASK_CMD + KEY_K);
```
Comment (homer666) [inline modules/visual_script/visual_script_editor.cpp]: Seems you left this line in from #28126

**Comment by Megalomaniak** [comment]
> This was in the original patch for VS, but there was some complaints about it. I don't remember the exact reasons.

At a guess, some might have wanted a contextual menu on right click, IMO should be an option for users to configure.

File: modules/visual_script/visual_script_editor.cpp (hunk 2 above)
Comment (Jummit) [reply modules/visual_script/visual_script_editor.cpp]: Fixed.

File: modules/visual_script/visual_script_editor.cpp (hunk 1 above)
Comment (Jummit) [reply modules/visual_script/visual_script_editor.cpp]: Nope, removed.

**Comment by Jummit** [comment]
What @Megalomaniak said could be done later, this is just to fix the issue I always faced where nodes would just spawn wherever they wanted.

**Comment by fire** [comment]
Should the existing node windows block the search?

**Comment by Jummit** [comment]
This isn't up to date anymore, since #28447.

**Comment by aaronfranke** [comment]
@Jummit Is this still desired? If so, it needs to be finished and rebased on the latest master branch.

If not, abandoned pull requests will be closed in the future as announced [here](https://godotengine.org/article/headsup-vulkan-merged-master-branch-unstable).

**Comment by Jummit** [comment]
I think I won't port it to master.

**Pull request closed without merging**